sourcegraph/cody
# Use correct file sources instead of always showing 'embeddings' in chat context

Fixes https://github.com/sourcegraph/cody/issues/2244

![Screenshot 2023-12-15 182047](https://github.com/sourcegraph/cody/assets/1078012/5ac9ddd3-ac74-49c2-a59e-15be3ca8444e)

![Screenshot 2023-12-15 182154](https://github.com/sourcegraph/cody/assets/1078012/02938d34-471c-4b19-abcd-78b7e0e390ad)

fyi @kalanchan 

## Test plan

- Select text in an editor and run a command like `/explain` that will include the context file from the editor
- Hover and ensure the tooltip says the file was from the editor, not embeddings
- Ask something that will cause the readme to be included automatically
- However and ensure the tooltip does not say embeddings
